enterprise/cmd/repo-updater/internal/authz/perms_syncer.go

```diff
@@ -628,12 +628,20 @@ func (s *PermsSyncer) syncUserPerms(ctx context.Context, userID int32, noPerms b
 		IDs:    map[int32]struct{}{},
 	}
 
+	unifiedResult := &database.SetPermissionsResult{
+		Added:   0,
+		Found:   0,
+		Removed: 0,
+	}
```

Comment:
```suggestion
	unifiedResult := &database.SetPermissionsResult{}
```

You can use the default values :)